libs/vkd3d/vkd3d_private.h

```diff
@@ -1569,12 +1569,22 @@ enum vkd3d_bindless_flags
     VKD3D_BINDLESS_RAW_SSBO     = (1u << 6),
 };
 
+enum vkd3d_bindless_set_flag
+{
+    VKD3D_BINDLESS_SET_SAMPLER  = (1u << 0),
```

Comment:
I like this approach. Does this mean for generic/mutable descriptor types that we will be able to have a single set with CBV | SRV | UAV | IMAGE | BUFFER ?

Reply:
Yes, that's the idea.